### ISSUE TITLE
Collect different tests from different folders and run it

### ISSUE THREAD

**Issue by terrasoff** [Codeception/Codeception#4884]
i have multiple (nested) modules and common bootstrap. I want to organize my application next way:

[root]
-- module1
--- services
---- tests
----- functional
----- unit
-- module2
--- services
---- tests
----- functional
----- unit

Main idea is to have tests in service namespace. So i try to do it like this
**root/codeception.yml**
```yml
include:
  - module1
  - module2
```

**root/module1/codeception.yml**
```yml
namespace: module1
include:
  - services/*/tests
settings:
  bootstrap: bootstrap.php # common bootstrap
  colors: false
```
I want this **common bootstrap** to be common for all services' tests, but i have to define this file for each suite. I've a lot of services and don't want to define bootstrap for each of them. 

Is there a possibility to set up common bootstrap (and modules as well) for all services tests? No to repeat configuration. Should be a way to collect different tests from different folders and run it.

**Comment by johannesschobel** [comment]
Hey @terrasoff .. did you manage to solve this? I am running into the same problem.. Can you take a look at my referenced issue?

It would be awesome if you could help me out with this!
Thanks a lot in advance,
Johannes

**Comment by terrasoff** [comment]
@johannesschobel no chances, the architecture of framework doesn't allow it, so the only thing you can do is to segregate it to [different applications](https://codeception.com/docs/08-Customization) with one runner

**Comment by johannesschobel** [comment]
Hey @terrasoff ,
thank you very much for answering that fast to my question.. I managed to resolve it by creating various `codeception` "projects" within my overall project.. Like the approach that is mentioned in your referenced `documentation file`.

However, i then came accross the issue, that i need the same "configuration" for both the `codeception.yml` and `*.suite.yml` files in each module. In order to overcome this issue, i added another `presetfile` attribute to both configuration files that allow "loading" another `yml` file from a different path.  The read values are then merged together!

I opened a PR #4984 that provides respective feature! Hope it gets merged, because it would help developers with such scenarios to remove duplicate configuration code! ;)

Thanks again for your help!
Cheers
